weaviate/batch/crud_batch.py

```diff
@@ -1047,6 +1053,9 @@ def create_references(self) -> list:
         weaviate.UnexpectedStatusCodeException
             If weaviate reports a none OK status.
         """
+        if self._batch_size is not None and self.dynamic:
+            _Warnings.manual_batching_when_dynamic_enabled()
+            return []
```

Comment:
I think that this line is a breaking change. It should be removed to preserve backwards compatibility